spec/std/int_spec.cr

```diff
@@ -315,6 +315,13 @@ describe "Int" do
     end
   end
 
+  describe "digits" do
+    it "return the digits of a number" do
+      123.digits.should eq([1, 2, 3])
```

Comment:
Might be worth adding more tests for other `Int` types, like `123_u8` or `123_u64`.

```cr
123_u64.digits.should be_a(Array(UInt64))
123_u64.digits.should eq([1, 2, 3])
```

Reply:
Done! 👍🏼